mne/report.py

```diff
@@ -1002,15 +1007,34 @@ class construction.
         comments : None | str | list of str
             A string of text or a list of strings of text to be appended after
             the figure.
+        replace : bool
+            Controls whether to replace figures in the report with the same
+            caption. Defaults to ``False``.
```

Comment:
this is not super clear for me. 

Maybe something like this.

`If True, figures already present that have the same caption will be replaced.`

I find it clearer.